lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.12.0-dev",
+  "version": "3.12.0-agents-and-basepaths-preview.0",
```

Comment:
are we gonna create a version like this? Should it be like this? `3.12.0-agents-and-basepaths.preview.0` instead?